CuriousKatie/Helper/Helper.swift

```diff
@@ -0,0 +1,157 @@
+        let interest24 = Interest(name: "photography", category: Category.art, gear: [Gear.photo], environment: Environment.anywhere)
+        let interest25 = Interest(name: "programming", category: Category.science, gear: [Gear.highTech], environment: Environment.indoor)
+        let interest26 = Interest(name: "rock climbing", category: Category.sport, gear: [Gear.protection], environment: Environment.outdoor)
+        let interest27 = Interest(name: "running", category: Category.sport, gear: [Gear.sportClothes, Gear.runningShoes], environment: Environment.outdoor)
+        let interest28 = Interest(name: "skiing", category: Category.sport, gear: [Gear.winterClothes, Gear.skiing], environment: Environment.outdoor)
+        let interest29 = Interest(name: "skydiving", category: Category.sport, gear: [Gear.flying], environment: Environment.outdoor)
+        let interest30 = Interest(name: "snowboarding", category: Category.sport, gear: [Gear.winterClothes, Gear.snowboarding], environment: Environment.outdoor)
+        let interest31 = Interest(name: "surfing", category: Category.sport, gear: [Gear.swimwear], environment: Environment.outdoor)
+        let interest32 = Interest(name: "swimming", category: Category.sport, gear: [Gear.swimwear], environment: Environment.anywhere)
+        let interest33 = Interest(name: "traveling", category: Category.social, gear: [Gear.backpack, Gear.photo], environment: Environment.outdoor)
+        let interest34 = Interest(name: "watching movies", category: Category.social, gear: [Gear.noGear], environment: Environment.indoor)
+        let interest35 = Interest(name: "writing", category: Category.art, gear: [Gear.noGear], environment: Environment.anywhere)
+        let interest36 = Interest(name: "yoga", category: Category.sport, gear: [Gear.sportClothes], environment: Environment.anywhere)
+        
+        return [interest01,
+                interest02,
+                interest03,
+                interest04,
+                interest05,
+                interest06,
+                interest07,
+                interest08,
+                interest09,
+                interest10,
+                interest11,
+                interest12,
+                interest13,
+                interest14,
+                interest15,
+                interest16,
+                interest17,
+                interest18,
+                interest19,
+                interest20,
+                interest21,
+                interest22,
+                interest23,
+                interest24,
+                interest25,
+                interest26,
+                interest27,
+                interest28,
+                interest29,
+                interest30,
+                interest31,
+                interest32,
+                interest33,
+                interest34,
+                interest35,
+                interest36]
```

Comment:
Just saw in you last commit 184f3d9f47344c6ba20be168da92e02f2775f130 that adding an interest made you rename a bunch of variables. An easy improvement is to simply removing them:
```
return [
   Interest(name: "acting", category: Category.art, gear: [Gear.noGear], environment: Environment.anywhere),
   Interest(name: "aerial acrobatics", category: Category.sport, gear: [Gear.sportClothes], environment: Environment.indoor),
   Interest(name: "astronomy", category: Category.science, gear: [Gear.optical, Gear.photo], environment: Environment.outdoor),
   Interest(name: "bike touring", category: Category.sport, gear: [Gear.ecoTransport], environment: Environment.outdoor),
   ...
]
```

Reply:
Done 😋